### PR TITLE
fix/tx-pending-modal Fix the Transaction pending modal

### PR DESCRIPTION
**Pull request by fqlx**
- Fix the Transaction pending modal on deposit and withdrawal
    - Added RPC node URI for avax networks

### CODE REVIEW

**Review by kenchangh — APPROVED** [review]

File: shared/src/hooks/web3Context.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+import { NODE_URI } from "../utils/env";
 
 export type Web3ContextData = {
   provider: BaseProvider;
 };
 
-const defaultProvider = ethers.getDefaultProvider(getNodeURI());
+const defaultProvider = ethers.getDefaultProvider(NODE_URI[parseInt(
+  (window as any)?.ethereum?.chainId
+)]);
```
Comment (kenchangh) [inline shared/src/hooks/web3Context.tsx:12]: Does this actually work if the user switches chain? This gets set at the start

**Review by kenchangh — CHANGES_REQUESTED** [review]
Just checking

File: shared/src/hooks/web3Context.tsx (hunk 1 above)
Comment (fqlx) [reply shared/src/hooks/web3Context.tsx:12]: It seems so but it's definitely not a recommended way to do it.

**Review by kenchangh — APPROVED** [review]

File: shared/src/hooks/web3Context.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 export type Web3ContextData = {
   provider: BaseProvider;
 };
 
-const defaultProvider = ethers.getDefaultProvider(getNodeURI());
+// TODO: Fix this in the future
+// Right now we just default to ETH mainnet for the unconnected wallet state
+// This means that we do not show the AVAX vault's details on the app
+const defaultProvider = ethers.getDefaultProvider(
+  NODE_URI[CHAINID.ETH_MAINNET]
```
Comment (fqlx) [inline shared/src/hooks/web3Context.tsx:14]: What about pulling this from `window.ethereum.chainId` and default to eth mainnet if it doesn't exist?

```
const defaultProvider = ethers.getDefaultProvider(
  NODE_URI[CHAINID.ETH_MAINNET]
);
```

**Pull request merged**